src/elements/content-preview/preview-header/Header.js

```diff
@@ -0,0 +1,129 @@
+/**
+ * @flow
+ * @file Preview header component
+ * @author Box
+ */
+
+import React from 'react';
+import { injectIntl } from 'react-intl';
+import PlainButton from '../../../components/plain-button/PlainButton';
+import IconClose from '../../../icons/general/IconClose';
+import IconDrawAnnotationMode from '../../../icons/annotations/IconDrawAnnotation';
+import IconPointAnnotation from '../../../icons/annotations/IconPointAnnotation';
+import IconPrint from '../../../icons/general/IconPrint';
+import IconDownload from '../../../icons/general/IconDownloadSolid';
+import AsyncLoad from '../../common/async-load';
+import messages from '../../common/messages';
+import { getIcon } from '../../common/item/iconCellRenderer';
+import { nines } from '../../../styles/variables';
```

Comment:
Changed to pull from the global vars file instead of a more local constants file.